iCubTelemVizServer/icubtelemetry.js

```diff
@@ -171,8 +171,8 @@ ICubTelemetry.prototype.generateTelemetry = function (timestamp,value,id) {
     switch(id) {
         case "sens.imu":
         case "sens.leftLegState":
-        case "sens.leftLegEEwrench":
-        case "sens.rightLegEEwrench":
+        case "sens.leftFootEEwrench":
+        case "sens.rightFootEEwrench":
```

Comment:
Wouldn't it be possible to also get this list directly from the ``json`` file?

Reply:
It's a good point, actually, I intend to simplify this function as well as the `update`. But I shouldn't need to save a field telling if the entry is already flat or not. It would be simpler to systematically call [`flatten`](https://github.com/ami-iit/yarp-openmct/blob/f2144093b7a6ea2b9fce1d0db7aab9a5bd33a63a/iCubTelemVizServer/icubtelemetry.js#L117) and do the check in there. Btw, I think it's already possible to do it without additional checks, I have to take a look.

Reply:
I see. My main point would be to simplify the logic for adding new ports. Having a single json file where you specify what you want would be already nice, but clearly this would be for future works

Reply:
Follow-up in https://github.com/ami-iit/yarp-openmct/issues/77 .